src/storage/disk_manager.c

```diff
@@ -1153,24 +1159,63 @@ disk_rv_undo_format (THREAD_ENTRY * thread_p, LOG_RCV * rcv)
 
   if (volid == disk_Cache->nvols_perm - 1)
     {
+      VPID vpid_volheader;
+      PAGE_PTR page_volheader = NULL;
+      DKNSECTS total = 0, max = 0;
+      DKNSECTS free = 0;
+
+      /* we need page to remove total/max from disk cache. however, it must be allocated. */
+      vpid_volheader.volid = volid;
+      vpid_volheader.pageid = DISK_VOLHEADER_PAGE;
+      ret = pgbuf_fix_if_not_deallocated (thread_p, &vpid_volheader, PGBUF_LATCH_READ, PGBUF_UNCONDITIONAL_LATCH,
+					  &page_volheader);
```

Comment:
I think page fix might be failed here. 
`disk_unformat` invalidates all pages of the volume and removes the volume file.
To fix the header if needed and then unformat would be safer.